docker-compose.yaml

```
@@ -0,0 +1,41 @@
+version: "3.3"
+
+services:
+  wordpress:
+    depends_on:
+      - db
+      - memcached
+    image: TODO
+    volumes:
+      - ./config:/app/config
+      - ./wp-content:/app/web/wp-content
+      - ./wp-config-docker-compose.php:/app/web/wp-config.php
+    ports:
+      - "8080:8080"
+    restart: always
+    environment:
+      DB_HOST: db:3306
+      DB_USER: wordpress
+      DB_PASSWORD: not-so-secure
+      DB_NAME: wordpress
+      MEMCACHED_HOST: memcached:11211
+      WP_HOME: http://localhost:8080
+      WP_SITEURL: http://localhost:8080/wp
+      WP_ENV: development
```

Local network is slower than cloud and 30s of timeout is not enough. 

```suggestion
      WP_ENV: development
      PHP_REQUEST_TIMEOUT: 60

```